App.tsx

```diff
@@ -164,14 +179,13 @@ export const App: React.FC<AppProps> = ({ navigation }) => {
                 <InfoListItem
                     title={'Emerson Field West'}
                     subtitle={['DG 100', 'EDR 5000', 'Online']}
-                    statusColor={PXBColors.red[500]}
-                    fontColor={PXBColors.red[500]}
+                    statusColor={theme.dark ? PXBColors.green[200] : PXBColors.green[500]}
+                    fontColor={theme.dark ? PXBColors.green[200] : PXBColors.green[500]}
                     hidePadding={true}
-                    backgroundColor={theme.dark ? PXBColors.darkBlack[100] : 'white'}
+                    backgroundColor={theme.colors.surface}
```

Comment:
What's the default background color for the InfoListItem? Is it not 'transparent'? Do we need to specify this at all?

Reply:
It is transparent, I'm ok leaving it transparent. I was just trying to match the DSM regardless of where the InfoList Item is located. Here's what it would look like outside of a 'surface' element with/without a set color.

![Simulator Screen Shot - iPhone 11 - 2021-03-11 at 10 02 01](https://user-images.githubusercontent.com/13989985/110807435-0a32df00-8251-11eb-80c8-9e33b578fd5a.png)
![Simulator Screen Shot - iPhone 11 - 2021-03-11 at 10 00 52](https://user-images.githubusercontent.com/13989985/110807437-0acb7580-8251-11eb-93f8-1abecdae9c55.png)

Reply:
That's what I would expect...it should blend in with whatever background it is sitting on. If user wants to override that they can with the background color (or change the background color of the thing they are sitting on).